sea-orm-codegen/src/entity/writer.rs

```diff
@@ -94,6 +95,14 @@ impl EntityWriter {
         self.entities
             .iter()
             .map(|entity| {
+                let entity_file = format!("{}.rs", entity.get_table_name_snake_case());
+                let column_names = entity
+                    .columns
+                    .iter()
+                    .map(|column| column.name.clone())
+                    .collect::<Vec<String>>();
+                info!("Generating {} {:?}", entity_file, column_names);
+
```

Comment:
I think we can show a bit more details of each column, e.g. https://github.com/kyoto7250/sea-orm/compare/issue_702...billy1624:pr/735
What's your thoughts?

Reply:
This will print something like...

```
Generating metadata.rs
	> Column `uuid`: Binary(Some(16)), not_null
	> Column `type`: String(Some(255)), not_null
	> Column `key`: String(Some(255)), not_null
	> Column `value`: String(Some(255)), not_null
	> Column `bytes`: Custom(BLOB), not_null
	> Column `date`: Date
	> Column `time`: Time(None)
Generating applog.rs
	> Column `id`: Integer(None), auto_increment, not_null
	> Column `action`: String(Some(255)), not_null
	> Column `json`: Json, not_null
	> Column `created_at`: Timestamp(None), not_null
```